#Copyright 2012 Cumulus Networks, Inc.  All rights reserved.

Reduce the log level of the oom_score_adj vs oom_adj usage warning

Both files exist in the kernel, but they are trying to move people to
oom_score_adj.  With that said, it's annoying for a warning to pop up
on normal boots.

diff --git a/fs/proc/base.c b/fs/proc/base.c
index d851316..cbcff3e 100644
--- a/fs/proc/base.c
+++ b/fs/proc/base.c
@@ -1076,7 +1076,7 @@ static ssize_t oom_adjust_write(struct file *file, const char __user *buf,
 	 * Warn that /proc/pid/oom_adj is deprecated, see
 	 * Documentation/feature-removal-schedule.txt.
 	 */
-	printk_once(KERN_WARNING "%s (%d): /proc/%d/oom_adj is deprecated, please use /proc/%d/oom_score_adj instead.\n",
+	printk_once(KERN_DEBUG "%s (%d): /proc/%d/oom_adj is deprecated, please use /proc/%d/oom_score_adj instead.\n",
 		  current->comm, task_pid_nr(current), task_pid_nr(task),
 		  task_pid_nr(task));
 	task->signal->oom_adj = oom_adjust;
